test/tests/multiapps/picard/tests

```diff
@@ -96,4 +96,14 @@
     issues = '#9038'
     requirement = "The system shall allow steady-state Picard iteration with pseudo-transient multiapps"
   [../]
+
+  [./steady_with_postprocessor_convergence]
+    type = 'Exodiff'
+    input = 'picard_custom_postprocessor.i'
+    exodiff = 'picard_custom_postprocessor_out.e picard_custom_postprocessor_out_sub0.e'
+    recover = false
+    design = 'multiapps/FullSolveMultiApp.md'
+    issues = '#14642'
+    requirement = "The system shall allow custom convergence check with user defined postprocessor."
```

Comment:
Remove `custom`. It will be part of the convergence check but not `custom` per se.